src/Microsoft.ML.StandardLearners/Standard/doc.xml

```diff
@@ -1,13 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <doc>
   <members>
-    
-    <member name="SDCA">
-      <summary>
-        Train an SDCA linear model.
-      </summary>
+    <!--  
+    The following text describes the SDCA algorithm details.
+    It's used for the remarks section of all SDCA-based trainers (binary, multiclass, regression)
+    -->
+    <member name="SDCA_remarks">
```

Comment:
> <member name="SDCA_remarks"> [](start = 3, length = 29)

so shall we keep the docs.xml for reuse, than?

Reply:
If we can reduce the usage to 1 by using cref links, we don't need doc.xml. We should use inline documentation.

I this case we have 4 SDCA trainers that all need to explain what SDCA is and we cannot cref them to each other. So I'm keeping doc.xml for it.

 

---
In reply to: [261051276](https://github.com/dotnet/machinelearning/pull/2771#discussion_r261051276) [](ancestors = 261051276)